doc/installation.md

```diff
@@ -424,6 +461,17 @@ Verify that the status shows nodes for BOTH Server 1 and Server 3
 ```
 # sup crossbar_maintenance create_account {ACCT NAME} {REALM} {LOGIN} {PASSWORD}
 ```
+### Load applications
+
+*Server 1*
+```
+# sup crossbar_maintenance init_apps /var/www/html/monster-ui/apps/ http://192.168.1.100:8000/v2
```

Comment:
Only need to run this once; if 2 crossbar servers are in use, should probably point this to an HAProxy to round-robin between crossbar servers instead. Or not, am ambivalent but it may be confusing?